test/jasmine/tests/transform_sort_test.js

```diff
@@ -275,6 +275,7 @@ describe('Test sort transform interactions:', function() {
         function hover(gd, id) {
             return new Promise(function(resolve) {
                 gd.once('plotly_hover', function(eventData) {
```

Comment:
Ah, this pattern is probably the reason we're timing out blind - when we wait for an event to resolve our promise, we need some timeout that causes an explicit error if that event never fires (and gets aborted when the event does fire), so we can see what's happening.

`delete gd._lastHoverTime` may well solve the issue here, but in case it doesn't (or something else crops up when we alter/extend this later) can you add that?